pagopa/pn-frontend
# Feat/pn 1827

## Short description
implemented dialog for view code and revoke delegate for g
## List of changes proposed in this pull request
- inser dialog component in deleghe page and delegatesBy company
- do test for delegatesByCompany component

## How to test
when you click in menu of row on the table you see a choose for view code or revoke
